packages/record-data/addon/-private/graph/index.ts

```diff
@@ -338,7 +338,7 @@ export class Graph {
 
   willDestroy() {
     this.identifiers.clear();
-    this.store = (null as unknown) as RecordDataStoreWrapper;
+    this.store = null as unknown as RecordDataStoreWrapper;
```

Comment:
Do you find this to be more readable?

Reply:
I don't find it better one way or the other, its just what prettier does now.